packages/client/src/features/tools/base-glsp-tool.ts

```diff
@@ -39,15 +39,15 @@ export abstract class BaseGLSPTool implements GLSPTool {
         return true;
     }
 
-    dispatchFeedback(actions: Action[], feedbackeEmitter?: IFeedbackEmitter): void {
-        this.feedbackDispatcher.registerFeedback(feedbackeEmitter ? feedbackeEmitter : this, actions);
-    }
-
     dispatchActions(actions: Action[]): void {
         this.actionDispatcher.dispatchAll(actions);
     }
 
-    deregisterFeedback(actions: Action[], feedbackeEmitter?: IFeedbackEmitter): void {
-        this.feedbackDispatcher.deregisterFeedback(feedbackeEmitter ? feedbackeEmitter : this, actions);
+    dispatchFeedback(actions: Action[], feedbackEmitter?: IFeedbackEmitter): void {
+        this.feedbackDispatcher.registerFeedback(feedbackEmitter ? feedbackEmitter : this, actions);
```

Comment:
Could be simplified to:
```suggestion
        this.feedbackDispatcher.registerFeedback(feedbackEmitter ?? this, actions);
```